src/fixedstring/fixedstring.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include <folly/FixedString.h>
+
+using namespace folly;
+
+int main() {
+  FixedString<10> str("abcdefghij");
```

Comment:
add the case of an empty string as well, and of a string with more than 30 characters